docs/sdk-docs/connecting/README.md

```diff
@@ -1,23 +1,106 @@
 
-If you want you can also use the Light Client SDK to connect to networks that are not officially supported.
+This environment uses:
 
-To connect for example on a privately deployed Raiden network,
-you can initialize the SDK as usual, and pass the Contract Info as the fourth parameter of the create function.
+- A specific version of [Raiden](https://github.com/raiden-network/raiden/commit/2e741dfdf4bfa564dec760abd5e3d8b2c9d30715)
+- A matrix transport server - `https://raidentransport.test001.env.raiden.network`
+- A PFS server - `https://pfs.raidentransport.test001.env.raiden.network`
+ 
+You can find the raiden version tagged on Docker Hub under `raidennetwork/raiden:testenv001`. To pull the image you need to run the following: 
 
-```typescript
-import { Raiden } from 'raiden-ts';
+```bash
+docker pull raidennetwork/raiden:testenv001
+```
+
+This transport does not participate in the matrix federation. For this reason, you have to explicitly specify it when starting raiden. You can use the following flag:
+
+```bash
+--matrix-server=https://raidentransport.test001.env.raiden.network
+``` 
+
+Similarly, you also have to specify the path-finding server:
+
+ ```bash
+--pathfinding-service-address https://pfs.raidentransport.test001.env.raiden.network
+```
+
+## Running a Raiden node in the test environment
+You can easily run a python node in the test environment by using docker. To get the supported Raiden version from `dockerhub` you need to run the following command:
+
+```bash
+docker pull raidennetwork/raiden:testenv001
+```
+
+The test environment uses the `Görli` testnet. For the purposes of this guide you assume that you a `geth` node runs locally on your computer. If you use a different ethereum client or RPC provider, please adjust accordingly.  
+
+```bash
+geth --goerli console --cache=512 --port 30303 --rpc --rpcapi eth,net,web3,txpool --rpccorsdomain "*" --rpcaddr "0.0.0.0"
+```
+
+### Using docker run
+You can start the container, by using the following command:
+
+```bash
+docker run --rm -it \
+    --network=host \
+    --mount src=/path/to/keystore,target=/keystore,type=bind \
+    raidennetwork/raiden:testenv001 \
+    --keystore-path /keystore \
+    --network-id 5 \
+    --environment-type development \
+    --eth-rpc-endpoint http://127.0.0.1:8545 \
+    --accept-disclaimer \
+    --matrix-server=https://raidentransport.test001.env.raiden.network \
+    --pathfinding-service-address https://pfs.raidentransport.test001.env.raiden.network \
+    --api-address "http://0.0.0.0:5001"
+```
+
+We use `--network=host` if the ethereum node runs locally on the host machine, to provide access to it from the container.  
+
+### Running the python client from source
+If you want to use raiden from source code you can start by cloning the raiden repository, and checkout the suggested commit:
+
+```bash
+git clone https://github.com/raiden-network/raiden
+cd raiden
+git checkout 2e741dfdf4bfa564dec760abd5e3d8b2c9d30715
```

Comment:
Same here about the pull requests, mentioned earlier

Reply:
The extra steps are only required if you have to build binaries. Using Raiden from a virtual environment doesn't require any additional steps.